samples/java-valueentity-customer-registry/src/main/java/customer/Main.java

```diff
@@ -25,12 +25,11 @@ public static AkkaServerless createAkkaServerless() {
       CustomerValueEntity::new,
       // end::register[]
       CustomerSummaryByNameView::new,
-      CustomerByEmailView::new,
       CustomersResponseByNameView::new,
+      CustomerByEmailView::new,
```

Comment:
So, here the names didn't change, but the ordering changed. Note that the `CustomerS` and `Customers` (plural). 

In any case, the ordering doesn't seen to be deterministic. 

CustomerByEmailView
CustomerActionImpl
CustomerByNameView

Alphabetical? Per component type? File system ordering?

Reply:
I created issue https://github.com/lightbend/akkaserverless-java-sdk/issues/407 for this.